### ISSUE TITLE
java.lang.RuntimeException: Unknown exception code: 1 msg null

### ISSUE THREAD

**Issue by Slake07** [Karumi/Dexter#251]
### Expected behaviour
I am getting this error on my release on play store for some users. I don't know the exact behavior.
Here are crash report:

Fatal Exception: java.lang.RuntimeException: Unknown exception code: 1 msg null
       at android.os.Parcel.readException(Parcel.java:1477)
       at android.os.Parcel.readException(Parcel.java:1419)
       at android.app.ActivityManagerProxy.checkPermission(ActivityManagerProxy.java:3516)
       at android.app.ContextImpl.checkPermission(ContextImpl.java:1770)
       at android.content.ContextWrapper.checkPermission(ContextWrapper.java:562)
       at androidx.core.content.PermissionChecker.checkPermission(PermissionChecker.java:97)
       at androidx.core.content.PermissionChecker.checkSelfPermission(PermissionChecker.java:134)
       at com.karumi.dexter.AndroidPermissionService.checkSelfPermission(AndroidPermissionService.java)
       at com.karumi.dexter.DexterInstance.isEveryPermissionGranted(DexterInstance.java)
       at com.karumi.dexter.DexterInstance.checkMultiplePermissions(DexterInstance.java)
       at com.karumi.dexter.DexterInstance.checkPermissions(DexterInstance.java)
       at com.karumi.dexter.Dexter.check(Dexter.java)
       at com.allbackup.ui.activity.InnerHomeActivity$setupListeners$2.onClick(HomeActivity.java:236)
       at android.view.View.performClick(View.java:4562)
       at android.view.View$PerformClick.run(View.java:18918)
       at android.os.Handler.handleCallback(Handler.java:808)
       at android.os.Handler.dispatchMessage(Handler.java:103)
       at android.os.Looper.loop(Looper.java:193)
       at android.app.ActivityThread.main(ActivityThread.java:5388)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:839)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:655)
       at dalvik.system.NativeStart.main(NativeStart.java)

### Version of the library
Version: 6.0.1

**Comment by pedrovgs** [comment]
Thanks @slake07! 

Do you know the volume of the occurrences? Can you share more information about the OS version where the bug is reproduced? What about the device model? It appears to be Android code throwing an error - leading us to believe it's either an Android bug or a bug in a certain flavors/forks of Android.

We'll investigate further and report back with any findings/plans to guard against this; any additional information regarding frequency and ability to reproduce would be greatly appreciated.

Thanks

**Comment by Slake07** [comment]
@pedrovgs 
It occurred only once. which was on Lenovo mobile. Don't know the model number. OS Version is 4.4.2. 

Another issue I also found with this library. should I create another bug or mention here only?

**Comment by pedrovgs** [comment]
Please, if the bug is not related to this one create a new issue.
